### PR TITLE
Test Automation Error-> to investigate on how to put unique data-testid on certain elements

### PR DESCRIPTION
**Pull request by sszafGCA**
# Description

## Links to Tickets or other pull requests
- https://ticketsystem.dbildungscloud.de/browse/BC-3002
- https://github.com/hpi-schul-cloud/e2e-system-tests/pull/120

## Approval for review

- [x] QA: In addition to review, the code has been manually tested (if manual testing is possible)
- [x] All points were discussed with the ticket creator, support-team or product owner. The code upholds all quality guidelines from the PR-template.

> Notice: Please remove the WIP label if the PR is ready to review, otherwise nobody will review it.

### CODE REVIEW

**Review by MateuszSapalaPassCon — APPROVED** [review]

File: static/scripts/administration.js [hunk 1]
```diff
@@ -138,6 +141,7 @@ $(document).ready(() => {
 			title: $t('administration.school.headline.addPolicy'),
 			closeLabel: $t('global.button.cancel'),
 			submitLabel: $t('global.button.add'),
+			submitDataTestId: 'add-modal--policy',
```
Comment (MartinSchuhmacher) [inline static/scripts/administration.js:144]: in general also here, please dont use mixture of hivens and double-hivens

File: views/courses/courseGroup.hbs [hunk 2]
```diff
@@ -140,7 +140,7 @@
 		{{$t "global.button.cancel"}}
 	</button>
 
-	<button type="submit" class="btn btn-primary btn-submit">
+	<button type="submit" class="btn btn-primary btn-submit" data-testid="delete-courseGroup-btn">
```
Comment (MartinSchuhmacher) [inline views/courses/courseGroup.hbs:143]:
```suggestion
	<button type="submit" class="btn btn-primary btn-submit" data-testid="delete-course-group-btn">
```

File: views/courses/edit-course.hbs [hunk 3]
```diff
@@ -268,7 +268,7 @@
 						</button>
 
                         {{> "lib/components/csrfInput"}}
-						<button type="submit" class="btn btn-primary btn-submit" data-testid="modal_delete_course_button">
+						<button type="submit" class="btn btn-primary btn-submit" data-testid="modal_edit_course_button">
```
Comment (MartinSchuhmacher) [inline views/courses/edit-course.hbs:271]:
```suggestion
						<button type="submit" class="btn btn-primary btn-submit" data-testid="modal-edit-course-button">
```

**Review by MartinSchuhmacher — CHANGES_REQUESTED** [review]

File: static/scripts/administration.js [hunk 4]
```diff
@@ -128,6 +130,7 @@ $(document).ready(() => {
 			title: $t('global.button.add'),
 			closeLabel: $t('global.button.cancel'),
 			submitLabel: $t('global.button.add'),
+			submitDataTestId: 'add-modal--rss',
```
[the remark this answers is not in the document]
Comment (sszafGCA) [reply static/scripts/administration.js]: fixed

File: static/scripts/administration.js (hunk 1 above)
Comment (sszafGCA) [reply static/scripts/administration.js]: fixed

File: views/courses/courseGroup.hbs (hunk 2 above)
Comment (sszafGCA) [reply views/courses/courseGroup.hbs]: fixed

File: views/courses/edit-course.hbs (hunk 3 above)
Comment (sszafGCA) [reply views/courses/edit-course.hbs]: fixed

File: static/scripts/administration.js [hunk 5]
```diff
@@ -141,7 +141,7 @@ $(document).ready(() => {
 			title: $t('administration.school.headline.addPolicy'),
 			closeLabel: $t('global.button.cancel'),
 			submitLabel: $t('global.button.add'),
-			submitDataTestId: 'add-modal--policy',
+			submitDataTestId: 'add-modal-Spolicy',
```
[the remark this answers is not in the document]
Comment (sszafGCA) [reply static/scripts/administration.js:144]: hahah yeah you are right

[the remark this answers is not in the document]
Comment (sszafGCA) [reply static/scripts/administration.js]: now it should be fine

**Review by MartinSchuhmacher — APPROVED** [review]

**Pull request merged**